src/auro-popover.js

```diff
@@ -158,6 +155,21 @@ export class AuroPopover extends LitElement {
     this.popper.show();
     this.isPopoverVisible = true;
     this.setAttribute('data-show', true);
+
+    this.handleMouseoverEvent();
+  }
+
+  /**
+   * Hides the popover when hovering outside of the popover or it's trigger.
+   * @private
+   * @returns {void}
+   */
+  handleMouseoverEvent() {
+    document.querySelector('body').addEventListener('mouseover', (evt) => {
+      if (!evt.composedPath().includes(this)) {
+        this.toggleHide();
+      }
+    });
```

Comment:
This eventListener needs to be removed after the popover is closed otherwise we are just going to overload the DOM with these.